src/components/AnonymousChat/child/ChannelImage.tsx

```diff
@@ -16,76 +17,88 @@ const ChannelImage = ({
   anonPostNotificationUserInfo = null,
   postMaker = null,
   isCommentExists = false,
-  isOwnSignedPost = false,
-  type = BaseChannelItemTypeProps.ANON_PM
+  type = BaseChannelItemTypeProps.ANON_PM,
+  isAnonymousTab = false
 }) => {
   const styles = StyleSheet.create({
     image: {
-      width: 48,
-      height: 48,
-      borderRadius: 24
+      position: 'relative',
+      width: dimen.normalizeDimen(48),
+      height: dimen.normalizeDimen(48),
+      marginTop: dimen.normalizeDimen(12),
+      marginBottom: dimen.normalizeDimen(12),
+      borderRadius: dimen.normalizeDimen(24)
     },
     postNotificationImage: {
-      width: 24,
-      height: 24,
-      borderRadius: 16,
       position: 'absolute',
-      top: 30,
+      top: dimen.normalizeDimen(30 + 12),
       right: 0,
+      width: dimen.normalizeDimen(24),
+      height: dimen.normalizeDimen(24),
+      borderRadius: dimen.normalizeDimen(12),
       borderWidth: 2,
       borderColor: colors.white,
       display: 'flex',
       justifyContent: 'center',
       alignItems: 'center'
     },
     myPostNotificationImageContainer: {
-      backgroundColor: colors.lightblue
+      backgroundColor: isAnonymousTab ? colors.bondi_blue : colors.darkBlue
     },
     anonPmNotificationImageContainer: {
       backgroundColor: colors.bondi_blue
     },
     postNotificationIcon: {
-      width: 12.5,
-      height: 12.5
+      width: dimen.normalizeDimen(12.5),
+      height: dimen.normalizeDimen(12.5)
+    },
+    chatIcon: {
+      width: dimen.normalizeDimen(12),
+      height: dimen.normalizeDimen(12)
+    },
+    backgroundDarkBlue: {
+      backgroundColor: colors.darkBlue
     }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The styles have been updated to use the `dimen.normalizeDimen` function for dimension values and added a new style `backgroundDarkBlue`. Ensure that these changes are reflected correctly in the UI and don't break the layout.
```diff
-      image: {
-       width: 48,
-       height: 48,
-       borderRadius: 24
-      },
-      postNotificationImage: {
-       width: 24,
-       height: 24,
-       borderRadius: 16,
-        position: 'absolute',
-       top: 30,
-        right: 0,
-        borderWidth: 2,
-        borderColor: colors.white,
-        display: 'flex',
-        justifyContent: 'center',
-        alignItems: 'center'
-      },
-      myPostNotificationImageContainer: {
-       backgroundColor: colors.lightblue
-      },
-      anonPmNotificationImageContainer: {
-        backgroundColor: colors.bondi_blue
-      },
-      postNotificationIcon: {
-       width: 12.5,
-       height: 12.5
-      }
+     image: {
+      position: 'relative',
+      width: dimen.normalizeDimen(48),
+      height: dimen.normalizeDimen(48),
+      marginTop: dimen.normalizeDimen(12),
+      marginBottom: dimen.normalizeDimen(12),
+      borderRadius: dimen.normalizeDimen(24)
+     },
+     postNotificationImage: {
+       position: 'absolute',
+      top: dimen.normalizeDimen(30 + 12),
+       right: 0,
+      width: dimen.normalizeDimen(24),
+      height: dimen.normalizeDimen(24),
+      borderRadius: dimen.normalizeDimen(12),
+       borderWidth: 2,
+       borderColor: colors.white,
+       display: 'flex',
+       justifyContent: 'center',
+       alignItems: 'center'
+     },
+     myPostNotificationImageContainer: {
+      backgroundColor: isAnonymousTab ? colors.bondi_blue : colors.darkBlue
+     },
+     anonPmNotificationImageContainer: {
+       backgroundColor: colors.bondi_blue
+     },
+     postNotificationIcon: {
+      width: dimen.normalizeDimen(12.5),
+      height: dimen.normalizeDimen(12.5)
+    },
+    chatIcon: {
+      width: dimen.normalizeDimen(12),
+      height: dimen.normalizeDimen(12)
+    },
+    backgroundDarkBlue: {
+      backgroundColor: colors.darkBlue
+     }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->